saleor/settings.py

```diff
@@ -81,16 +86,23 @@ def get_bool_from_env(name, default_value):
     ('vi', _('Vietnamese')),
     ('zh-hans', _('Chinese')),
     ('zh-tw', _('Chinese (Taiwan)'))]
+
+_tmp = env('SALEOR_LANGUAGES')
+if _tmp:
```

Comment:
Please use meaningful variables

Reply:
sorry my english, meaningful variables - you mean what _tmp variable must have more explained name? for example _tmp_saleor_languages?
i good understand? fixed